### ISSUE TITLE
There is no option for anonymous login in the Supabase Package.

### ISSUE THREAD

**Issue by Thaddeus19** [supabase/supabase-flutter#983]
**Describe the bug**
When I try to set up the flow for my application to use anonymous users, I enable the options in the project settings, but when I try to call `const { data, error } = await supabase.auth.signInAnonymously();` , it returns the error "Property 'signInAnonymously' does not exist on type 'SupabaseAuthClient'."

I have already checked all the methods that can be used with 'SupabaseAuthClient', but the Anonymous one does not appear.

<img width="636" alt="Captura de pantalla 2024-07-22 a la(s) 10 56 04 a m" src="https://github.com/user-attachments/assets/fcf8c337-d6a4-4f5b-99e3-5cf5ea1da783">

**Comment by arrrrny** [comment]
this is the flutter package and your code is javascript. 
<img width="430" alt="Screenshot 2024-07-23 at 8 50 37 AM" src="https://github.com/user-attachments/assets/71578f7b-e43b-4211-b438-4a912df2f156">


I have signInAnonymously on flutter and works fine. need to setup config.toml as 

```
[auth]
enabled = true
enable_anonymous_sign_ins = true
```

**Comment by Vinzent03** [comment]
Yeah this is the flutter package and not the JavaScript version, which I guess you are searching for. But it exists in js as well https://supabase.com/docs/reference/javascript/auth-signinanonymously . You probably need to update your packages.
